R/get_pvol_se.R

```diff
@@ -0,0 +1,43 @@
+get_pvol_se_radar_mapping <- c(
+  "seang" = "angelholm", "seatv" = "atvidaberg", "sebaa" = "balsta",
+  "sehem" = "hemse", "sehuv" = "hudiksvall", "sekaa" = "karlskrona",
+  "sekrn" = "kiruna", "selek" = "leksand", "sella" = "lulea",
+  "seoer" = "ornskoldsvik", "seosd" = "ostersund", "sevax" = "vara"
+)
+
+get_pvol_se <- function(radar, time, ..., call = rlang::caller_env()) {
+  if (!radar %in% names(get_pvol_se_radar_mapping)) {
+    cli::cli_abort(
+      c(
+        x = "The radar {.val {radar}} is not found in the mapping for Swedish radars",
+        i = "Most likely this means an invalid odim code has been provided"
+      ),
+      call = call, class = "getRad_error_get_pvol_se_radar_not_found"
+    )
+  }
+  radar_name <- get_pvol_se_radar_mapping[radar]
+  url <- glue::glue('/area/{radar_name}/product/qcvol/{lubridate::year(time)}/{lubridate::month(time)}/{{lubridate::day(time)}}/radar_{radar_name}_qcvol_{strftime(time, "%Y%m%d%H%M", tz="UTC" )}.h5')
+  req <- withCallingHandlers(
```

Comment:
withCallingHandlers is new to me, why use it over trycatch or even purrr adverbs? 

https://purrr.tidyverse.org/reference/safely.html

Reply:
Because I only wanted to modify the `404` errors and make those more clear, I followed the suggestion here: https://httr2.r-lib.org/reference/req_error.html